### ISSUE TITLE
[Adaptive Card: info icon]: Screen reader is not reading information that appears after expanding information icon on 'Public adaptive card schema' Preview.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7011]
https://microsoft.visualstudio.com/OS/_workitems/edit/36485981

**Comment by jwoo-msft** [comment]
This issue seems to invalid test case.
1. Card is not expanded correctly due to missing data

**Comment by PankajBhojwani** [comment]
One potential solution for this issue is to tell the `ToggleVisibility` action to move focus to the element that it just set to `Visible`. However, this potentially breaks situations where the `ToggleVisibility` action is actually being used to toggle the visibility of several items such as in https://microsoft.visualstudio.com/OS/_workitems/edit/38275351 (instead of being used to 'open up a new control'). 

Looking deeper, this issue, #6264, #6274 all stem from the same underlying problem: 

_Users want an expander control_, but their only option right now is to simulate an expander by using a `button` with the `ToggleVisibility` action on it.

After some discussion, @carlos-zamora and I are thinking the best way to solve these issues is to implement a new control that is truly an expander. Then, that control can do the following:

- Update its own image and alt text when expanded/unexpanded (solving #6274)
- Move focus to its 'expanded content' when it gets expanded (solving this issue and #6264)

@jonmill @paulcam206, I've set up a meeting with all of us to discuss this!

**Comment by anna-dingler** [comment]
@jwoo-msft @paulcam206  Should we close this since ToggleVisibility is deprecated?

**Comment by anna-dingler** [comment]
Closing this issue because it is caused by ToggleVisibility which is a deprecated control.
